src/test/java/teammates/test/cases/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -59,6 +59,23 @@ public void testNewQuestionFrame() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("MCQ");
         assertTrue(feedbackEditPage.verifyNewMcqQuestionFormIsDisplayed());
+
+        feedbackEditPage.clickGenerateMcqOptionsCheckbox(-1);
+        assertFalse(feedbackEditPage.isElementVisible("mcqChoiceTable--1"));
+
+        feedbackEditPage.clickDiscardChangesLinkForNewQuestion();
+        feedbackEditPage.waitForConfirmationModalAndClickOk();
+
+        ______TS("MCQ: Check UI after cancelling");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("MCQ");
+        assertTrue(feedbackEditPage.verifyNewMcqQuestionFormIsDisplayed());
+        assertFalse(feedbackEditPage.isElementVisible("mcqChoiceTable--1"));
+        assertTrue(feedbackEditPage.isElementEnabled("mcqGenerateForSelect--1"));
+
+        feedbackEditPage.clickGenerateMcqOptionsCheckbox(-1); //reset back to normal condition
```

Comment:
Do we have a constant (might be defined somewhere) for the magic `-1`? (I know it is the index of a new question, but it is not really friendly to humans). Besides, you can also try to improve the comment. `Make the checkbox unchecked` might be better than `reset back to normal condition`